Scripts/script-SendEmailOnSLABreach.yml

```diff
@@ -0,0 +1,24 @@
+fromversion: 4.1.0
```

Comment:
Couple of questions, why is it from 4.1.0? as i don't see any special feature here?
Where is it used?
Shouldn't you get arguments for this script instead of sending mails to `someuser@demisto.com`?

Reply:
It's 4.1 because SLA field is a new feature that will only be available starting in 4.1.
As for the arguments, you're right - will be fixed.